monkey/tests/unit_tests/monkey_island/cc/resources/test_agents.py

```diff
@@ -17,8 +21,16 @@
 }
 
 
+@pytest.fixture
+def flask_client(build_flask_client):
```

Comment:
Python (or pytest) will use the nearest fixture if there are multiple with the same name?

Reply:
Yep!